### ISSUE TITLE
[FEATURE] Support for sftp!

### ISSUE THREAD

**Issue by markaltmaier** [nodeftpd/nodeftpd#154]
Is it possible for this package to support sftp instead of ftp?!

**Comment by mk-pmb** [comment]
If you're refering to the [SSH File Transfer Protocol](https://en.wikipedia.org/wiki/SSH_File_Transfer_Protocol), we will probably never add that, because it has nothing to do with FTP except in name.

**Comment by Mustafa-Agha** [comment]
Yeah, I was talking about this package ssh2-sftp-client, so ftp uses a lot of ports, and it's difficult to support through tunneling. On the other hand, sftp uses only one port, 22, which can be supported through tunneling.

**Comment by mk-pmb** [comment]
Yes, indeed, nowadays, FTP is no longer a good choice for most scenarios. Albeit I'm helping to keep this project alive, it is not something I use on a regular everyday basis. FTP has become a niche solution for exotic scenarios.
